frontend/www/js/omegaup/components/problem/CollectionDetails.test.ts

```diff
@@ -10,11 +10,27 @@ describe('CollectionDetails.vue', () => {
     const wrapper = shallowMount(collection_Details, {
       propsData: {
         data: {
-          type: 'author',
+          type: 'problemLevelBasicIntroductionToProgramming',
+          collection: <string[]>[
+            'problemTagMatrices',
+            'problemTagDiophantineEquations',
+            'problemTagInputAndOutput',
+            'problemTagArrays',
+          ],
+          anotherTags: <string[]>[
```

Comment:
Este nombre ya cambió por:
```suggestion
          publicTags: <string[]>[
```